javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,150 @@
+
+   let names =[];
+   let arrTodo=[];
+
+   function checkName(name){
+       for(let i=0;i<names.length;i++){
+           if(names[i]==name)
+           return true;
+           else
+           return false;
+
+       }
+   }
+
+
+function getReply(command){
+
+    if(typeof(command)!= "string"){
+        console.log('Invalid Command')
+        return
+       // return'command not valid';      
+       }
+       
+   
+    if (command.includes('Hello my name is')){
+       name=command.slice(17,command.length);
```

Comment:
There is an alternative way as below example by converting to array of words

 const commandWords = inputString.split(' ');

  if (inputString.startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }